<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsPointClusterRendererWidgetBase</class>
 <widget class="QWidget" name="QgsPointClusterRendererWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="1">
    <widget class="QComboBox" name="mRendererComboBox"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="mDistanceToleranceLabel">
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="mRendererSettingsButton">
     <property name="text">
      <string>Renderer Settingsâ€¦</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QgsDoubleSpinBox" name="mDistanceSpinBox">
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsUnitSelectionWidget" name="mDistanceUnitWidget" native="true">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mRendererLabel">
     <property name="text">
      <string>Renderer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mCenterSymbolLabel">
     <property name="text">
      <string>Cluster symbol</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsSymbolButton" name="mCenterSymbolToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSymbolButton</class>
   <extends>QToolButton</extends>
   <header>qgssymbolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsUnitSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsunitselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mCenterSymbolToolButton</tabstop>
  <tabstop>mRendererComboBox</tabstop>
  <tabstop>mRendererSettingsButton</tabstop>
  <tabstop>mDistanceSpinBox</tabstop>
  <tabstop>mDistanceUnitWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
